src/PyodideWorker.js

```diff
@@ -260,7 +260,12 @@ const PyodideWorker = () => {
           return;
         }
       },
-      after: () => {},
+      after: () => {
+        pyodide.runPython(`
+        import matplotlib.pyplot as plt
+        plt.clf()
+        `);
```

Comment:
❓ it makes sense but I also thought anything without a `_` got wiped? I'm just surprised `pyplot` made it through 🤔 weird

Reply:
Hmmm that's true, I'm not sure - I'd assumed it was because matplotlib isn't getting reloaded, and somehow it remembered what the graph was meant to be... 🤷‍♀️

Reply:
I can dig deeper though if you like... 🤔

Reply:
na its ok, I think we might just need to keep this sort of thing in mind, no everything is wiped and it can lead to weird behaviour is all

Reply:
Okay, I just had a look and it appears that `plt` itself isn't making it through if you run again without defining it... So it must just remember what was in the plot another way and then be putting it back when they try to make a new graph. Seems odd though, like you say